documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -141,6 +141,62 @@ env:
 <3> The keystore contains the private key for mTLS authentication.
 <4> The password for accessing the keystore.
 
+. If you need a custom `SASL` configuration. For example, to access a cluster in a cloud provider with a custom login module like `AWS_MSK_IAM`. The `STRIMZI_SASL_CUSTOM_CONFIG_JSON` is the mechanism to do that.
+The property `STRIMZI_ALTERABLE_TOPIC_CONFIG` defaults to `ALL`. When a subset of properties is alterable, use a comma separated list, as below. In the case that a managed Kafka service doesn't allow any `.spec.config` properties to be set in the `KafkaTopic` a value of `NONE` is equivalent to an empty property.
+Otherwise, a comma separated list of the properties that can be altered is used.
```

Comment:
Nit: we have this convention of one sentence per line, which helps to make the doc review easier. It will still be rendered as before.

```suggestion
. If you need a custom `SASL` configuration. For example, to access a cluster in a cloud provider with a custom login module like `AWS_MSK_IAM`.
The `STRIMZI_SASL_CUSTOM_CONFIG_JSON` is the mechanism to do that.
The property `STRIMZI_ALTERABLE_TOPIC_CONFIG` defaults to `ALL`.
When a subset of properties is alterable, use a comma separated list, as below.
In the case that a managed Kafka service doesn't allow any `.spec.config` properties to be set in the `KafkaTopic` a value of `NONE` is equivalent to an empty property.
Otherwise, a comma separated list of the properties that can be altered is used.
```